helm/chart/values.yaml

```diff
@@ -14,7 +14,7 @@ controller:
   isDefaultIngressClass: true
   image:
     repository: traefik
-    tag: "v2.8"
+    tag: "v2.10"
```

Comment:
💡 

@zyassine 

Similarly to Argo internal chart, would it make sense to only maintain the app version directly in the chart rather than onto its deployment in TF for simplicity?

Reply:
Yes good idea !